src/styles/styles.css

```diff
@@ -151,3 +151,13 @@ h2 {
 .address-card h2 {
   font-size: 1.15rem;
 }
+
+.makeInlineBlock {
+  display: inline-block;
+  padding-left: 0.5em;
+}
+
+.text-link {
+  color: inherit;
```

Comment:
Not seen this rule before. So 'inherit' means it takes the value from the parent.